source/includes/fundamentals/code-examples/crud/change/Replace.cs

```diff
@@ -0,0 +1,72 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using static System.Console;
+
+namespace CSharpExamples.Fundamentals.ReplaceOne;
+
+public class ReplaceOne
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private const string MongoConnectionString = "<Your MongoDB URI>";
+
+    public static void Main(string[] args)
+    {
+        Setup();
+
+        // Create filter 
+        var filter = Builders<Restaurant>.Filter.Eq(r => r.Name, "Pizza Town");
```

Comment:
[suggestion]
All the code examples for Update use `restaurant` instead of `r` when building the filter query. Maybe change these to `restaurant` for consistency and better readability?

Reply:
This applies to other lines with `Filter.Eq` as well, but I did not tag them all in the code example.